iphone/Classes/TiUIWindowProxy.m

```diff
@@ -800,6 +800,32 @@ - (void)setTitle:(NSString *)title_
       [NSThread isMainThread]);
 }
 
+#ifdef __IPHONE_11
```

Comment:
There is no macro __IPHONE_11 . It should be  __IPHONE_11_0.
Better is we use our own defined macro IS_XCODE_9.

Reply:
Agree!